_data/devdocs/en/guides/payment_processing.md

```diff
@@ -153,16 +153,18 @@ You must pay by: 2014-04-01 at 23:00 UTC
 
 Indicating the denomination is critical. As of this writing, popular
 Bitcoin wallet software defaults to denominating amounts in either bitcoins (BTC)
-, millibitcoins (mBTC) or microbitcoins (uBTC, "bits"). Choosing between each unit is widely supported,
+, millibitcoins (mBTC) or microbitcoins (uBTC, "bits").
+Choosing between each unit is widely supported,
 but other software also lets its users select denomination amounts from
-some or all of the following options:
+some preselected or all [possible options](https://en.bitcoin.it/wiki/Units):
```

Comment:
The wording here seems to imply the table below includes all options, which is far from complete.

Reply:
(This PR is probably an *improvement* over the status quo, however)

Reply:
also thought of adding all decimals to the table, but wasnt sure thats helpful, is it? Or what wording would be more appropriate to not incline the table below includes all possibilities?